### PR TITLE
Fix empty publish (switch to NPM)

### PR DESCRIPTION
**Pull request by tpluscode**
Superseeding #50, moving from `yarn` to `npm` should fix the accidental publishing problem

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -4,30 +4,30 @@ on:
...
 
       - name: Setup Node.js
-        uses: actions/setup-node@v2.1.2
+        uses: actions/setup-node@v4
         with:
-          node-version: 14
+          node-version: 20
 
       - name: Install Dependencies
         run: yarn
```
Comment (matthieubosquet) [inline .github/workflows/release.yml:27]:
```suggestion
          node-version: 20

      - name: Install Dependencies
        run: npm i
```

Don't we need to also amend this?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -4,30 +4,30 @@ on:
...
 
       - name: Create Release Pull Request or Publish to npm
-        id: changesets
-        uses: changesets/action@master
+        uses: changesets/action@v1
         with:
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
           publish: yarn release
```
Comment (matthieubosquet) [inline .github/workflows/release.yml:33]:
```suggestion
        uses: changesets/action@v1
        with:
          # This expects you to have a script called release which does a build for your packages and calls changeset publish
          publish: npm release
```

And this?

**Review by matthieubosquet — COMMENTED** [review]
Do you think maybe the `yarn` command needs to be removed from the action steps to truly move away from Yarn?

File: .github/workflows/release.yml (hunk 1 above)
Comment (tpluscode) [reply .github/workflows/release.yml:27]: Of course!

**Review by matthieubosquet — APPROVED** [review]
LGTM

**Review by rubensworks — APPROVED** [review]

**Pull request merged**